src/app/pages/system/update/update.component.html

```diff
@@ -95,33 +95,35 @@
 </mat-card>
 
 <mat-card *ngIf="updates_available && pre_release_train && isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i><strong>This is not a production release, and should only be used for testing.</strong></div>
-  <div class="stable-warning" id="stable_warning_line_2">Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>
+    <strong>{{"This is not a production release, and should only be used for testing." | translate}}</strong>
+  </div>
+  <div class="stable-warning" id="stable_warning_line_2">{{"Before updating, please read the
+    <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">release notes</a>." | translate}}
   </div>
 </mat-card>
 
 <mat-card *ngIf="updates_available && release_train && isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/knowledgebase_category/freenas/?latest=true' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>{{"Before updating, please read the
```

Comment:
as above

Reply:
Here I don't see a technical reason other then URL appearing in translated string. But I saw that practice widely in the translation already.

Reply:
this was an error, sorry